src/components/UserProfile.re

```diff
@@ -161,6 +170,24 @@ module UserDetails = {
             href={"https://" ++ etherScanUrl ++ "/address/" ++ userAddress}>
             {Helper.elipsify(userAddress, 10)->restr}
           </a>
+          <br />
+          {isAddressCurrentUser || true
```

Comment:
`|| true` means that this condition is always true.
We should add a comment above this saying something like:
```
// NOTE: the number of loyalty tokens of a user currently will always show.
//       We are thinking of making this "private" only to the current logged in user. To enable this remove the `|| true` from the line below.
```